### PR TITLE
PLAT-122626: Fix Spinner to pause when `paused` prop is true

### PR DESCRIPTION
**Pull request by adrian-cocoara-lgp**
Fixed bug for pausing the animation of Spinner when 'paused'=true

Enact-DCO-1.0-Signed-off-by: Adrian Cocoara adrian.cocoara@lgepartner.com

### CODE REVIEW

**Review by daniel-stoian-lgp — APPROVED** [review]

**Review by stanca-pop-lgp — APPROVED** [review]

File: Spinner/Spinner.module.less [hunk 1]
```diff
@@ -140,6 +140,12 @@
 				border-radius: @agate-spinner-searching-border-radius;
 			}
 
+			&.pausedAnimation {
+				.node {
+					animation-play-state: paused;
+				}
+			}
+
```
Comment (ybsung) [inline Spinner/Spinner.module.less:148]: How about defining `&.pausedAnimation` once for `searching` and `loading`?
```
		&.searching,
		&.loading {
			&.pausedAnimation {
				.node {
					animation-play-state: paused;
				}
			}
```

**Comment by ybsung** [comment]
Hello @adrian-cocoara-lgp 
Could you update [CHANGELOG.md](https://github.com/enactjs/agate/blob/develop/CHANGELOG.md)?

**Comment by adrian-cocoara-lgp** [comment]
Hello @ybsung,
CHANGELOG.md is updated, and I applied your suggestion. Thanks!

**Comment by ybsung** [comment]
Hi @adrian-cocoara-lgp 
```
when 'paused' prop is 'true'
```
In your updated CHANGELOG.md,
the prop name should be wrapped with `` not '' and true value should not be wrapped with ''. I fixed it.

**Review by ybsung — APPROVED** [review]
LGTM

File: Spinner/Spinner.js [hunk 2]
```diff
@@ -159,14 +169,16 @@ const SpinnerBase = kind({
 	},
 
 	computed: {
-		className: ({children, color, size, transparent, styler}) => styler.append(
+		className: ({children, color, paused, size,  styler, transparent}) => styler.append(
```
Comment (jeonghee27) [inline Spinner/Spinner.js:172]:
```suggestion
		className: ({children, color, paused, size,  styler, transparent}) => styler.append(
```
```suggestion
		className: ({children, color, paused, size, styler, transparent}) => styler.append(
```
two space between size and styler

**Comment by jeonghee27** [comment]
Please resolve conflicts

**Pull request merged**